tools/rating_curve_comparison.py

```diff
@@ -238,6 +239,9 @@ def generate_rating_curve_metrics(args):
         else:
             generate_facet_plot(rating_curves, rc_comparison_plot_filename, nwm_recurr_data_table)
 
+        if single_plot:
```

Comment:
Instead of doing it this way where it will produce both facet plots and single gage plots, I think we should do single plots OR the default facet plots. Something like this:
```
        # plot rating curves
        if alt_plot:
            generate_rc_and_rem_plots(rating_curves, rc_comparison_plot_filename, nwm_recurr_data_table, hydrotable_filename)
        elif single_plot:
            generate_single_plot(rating_curves, rc_comparison_plot_filename, nwm_recurr_data_table)
        else:
            generate_facet_plot(rating_curves, rc_comparison_plot_filename, nwm_recurr_data_table)

```